components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -111,6 +115,9 @@
 @PowerMockIgnore("java.sql.*")
 public class SCIMUserManagerTest extends PowerMockTestCase {
 
+    public static final String DISPLAY_NAME_LOCAL_CLAIM = "http://wso2.org/claims/displayName";
+    public static final String TENANT_DOMAIN = "carbon.super";
```

Comment:
You can use `org.wso2.carbon.utils.multitenancy.MultitenantConstants.SUPER_TENANT_DOMAIN_NAME` instead of defining a new constant.